repl/src/test/scala/com/cloudera/livy/repl/PythonInterpreterSpec.scala

```diff
@@ -150,6 +149,7 @@ class PythonInterpreterSpec extends BaseInterpreterSpec {
       "name 'x' is not defined",
       List(
         "Traceback (most recent call last):\n",
+        "  File \"<stdin>\", line 1, in <module>\n",
```

Comment:
Changed the clean up. Traceback of
```python
def func1():
  raise Exception("message")
def func2():
  func1()
func2()
```
will become
```python
Traceback (most recent call last):
  File "<stdin>", line 4, in func2
  File "<stdin>", line 2, in func1
Exception: message"
```

I don't want to do too many changes at once and will leave line numbers as is. We can revisit line number later.